apps/showcase/src/assets/trainings/sdk/steps/introduction/instructions.html

```diff
@@ -0,0 +1,41 @@
+<h3>Why an SDK?</h3>
+<p>
+  If an application contains an API, there are several developer tasks that are required including a lot of redundant code.
+  These tasks include:
+</p>
+<ul>
+  <li>Choosing the needed techniques to perform API calls</li>
+  <li>Determining which HTTP method to use for each request</li>
+  <li>Managing the Authorization (maybe adding request headers) and do it for each request</li>
+  <li>Preparing the URL with any path or query parameters expected by the API endpoint</li>
+</ul>
+<p>
+  Let's see how we use the OpenAPI tools to handle our APIs and generate a typescript SDK codebase.
+  Check out the following diagram to visualize the description below.
+</p>
+<figure class="figure">
+  <img src="/assets/sdk-training-openapi-diagram.svg" width="75%" class="figure-img img-fluid mx-auto d-block">
```

Comment:
it seems weird to use `width` with `img-fuild`, maybe you should use padding instead